test/edu/macalester/comp127/hw0/AreaTest.java

```diff
@@ -10,11 +10,11 @@
 public class AreaTest {
     @Test
     public void testSquare() {
-        assertEquals(9.0, Area.getSquareArea(3.0), 0.001);

```

Comment:
Be careful not to modify the tests when the instructions say not to! We won't take a point off for this this time.